fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -492,6 +494,13 @@
     @VariableMgr.VarAttr(name = SINGLE_NODE_EXEC_PLAN, flag = VariableMgr.INVISIBLE)
     private boolean singleNodeExecPlan = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_HIVE_COLUMN_STATS, flag = VariableMgr.SESSION)
```

Comment:
flag = VariableMgr.SESSION is unnecessary.

Reply:
done